### ISSUE TITLE
Control subtitle/caption line position/margin

### ISSUE THREAD

**Issue by lwcorp** [howardchung/watchparty#511]
Subtitles/Captions are currently shown at the very bottom, which means they get cut off in some TVs.

Since you convert subtitles/captions to WebVTT anyway (since that's what `<video><track>` supports), you might as well add a line position setting, which is a feature of VTT that can be controlled with JavaScript.

Basically every time a VTT track is loaded, which you can track with `track.addEventListener('load', ...)`, then you can loop it and give it an alternative `line` position, [like this](https://stackoverflow.com/a/72807244/4829915).

**Comment by howardchung** [comment]
Yeah that's the correct function. But this would definitely need some UI/advanced options menu to supply the input. And I think we would need to network that value as well so all clients see it correctly.

**Comment by lwcorp** [comment]
In [my own version of watchparty](https://github.com/lwcorp/watchparty) (not related to yours), I just have a number input field which changes the line margin live (it defaults to -3 to be 3 lines from the bottom, thus not covered in my TV):

https://user-images.githubusercontent.com/1773306/215263876-f44327af-e9c9-41c9-8a72-28498481c39b.mp4

**Comment by howardchung** [comment]
I recently redesigned the subtitle modal so there's room to add more options now. But in the meantime maybe it would be quick and easy to add a static non-customizable offset value
